base/src/org/compiere/model/MProduction.java

```diff
@@ -44,7 +44,6 @@
 import org.compiere.util.Env;
 import org.compiere.util.Msg;
 import org.compiere.util.Util;
-import org.eevolution.manufacturing.model.MPPProductBOMLine;
```

Comment:
Please, this pull request should be atomic. A production class is unnecessary for this scope.

Can you use a fresh branch?

Reply:
Sure, I checkout master and pick this commits again without this change